api-tests/src/test/resources/org/openmrs/module/reporting/include/ReportTestDataset-encounter-before-midnight.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
@mseaton I'm particularly unsure about this part. One test, `evaluate_shouldFindPatientsWithEncountersCreatedOnTheSpecifiedDateIfPassedInTimeIsAtMidnight` depended on the ability to change the `date_created` of an existing encounter. That doesn't seem to be possible in 2.4, so instead I created this file to load an encounter that would get that test to pass without having to change a large number of other tests (which would be the case if I modified `ReportTestDataset-openmrs-2.4.xml` directly for example.

Reply:
Works for me - seems good.

Reply:
The alternative also would be to not use a test dataset for this, but to just use the PatientBuilder classes that you've seen used in other tests to build the data programmatically as needed.